bin/beaver-install

```diff
@@ -16,7 +17,13 @@ do
         shift
         ;;
         -d)
-        export BEAVER_CONTEXT_DIR="$2"
+        beaver_context_dir="$2"
+        shift
+        shift
+        ;;
+        -I)
+        dockerfile_commands+=("-I")
+        dockerfile_commands+=("$2")
```

Comment:
I wonder if it makes sense to forward arguments to `docker gen-dockerfile` manually like this of if it wouldn't make more sense to have an option to just forward any argument to it, like `dmd`s `-L` forwards arguments to the linker. So: `-I-Idocker/builder` will forward `-Idocker/builder` to `gen-dockerfile`.

Reply:
Can work. Right now I am trying to flesh out a bit more generic arg parsing library function to be reused from different beaver command so that everything works in a uniform way.